rules/windows/process_creation/proc_creation_win_susp_runonce_execution.yml

```
@@ -19,8 +20,14 @@ detection:
         - Image|endswith: '\runonce.exe'
         - Description: 'Run Once Wrapper'
     selection_cli:
-        CommandLine|contains: ' /AlternateShellStartup'
-    condition: all of selection*
+        CommandLine|contains: 
+            - '/AlternateShellStartup'
+            - '/r'
```

I think to avoid FP we do an `endswith` here and then we can remove the filter.
```suggestion
        - CommandLine|contains: '/AlternateShellStartup'
        - CommandLine|endswith: '/r'
```